README.md

```diff
@@ -2,12 +2,9 @@
 
 This repo is a collection of code supporting the Wiki Loves Monuments competition on the German Wikipedia.
 
-At the moment it mostly consists of bots that analyze and prepare the monument pages. 
-See the [bots README](update-bot/README.md) for detailed documentation on the bots.
+At the moment it consists of bots that analyze and prepare the monument pages and a PHP script that redirects from Wikipedia links to the Commons Uploader.
 
-## Installation
 
-Python 2
+See the [bots README](update-bot/README.md) for detailed documentation on the bots.
 
-    python -m pip install -r requirements.txt
-    python -m pip install -r dev-requirements.txt
+See the [redirect script README](redirect_script/README.md) for detailed documentation on the redirect scripts.
```

Comment:
There seems to be some confusion with the name of the PHP component. Here it called a redirect script while elsewhere it is a forward script. Note that the reference to the README file above is currently broken due to the incorrect directory name.